homeassistant/components/sesame/lock.py

```diff
@@ -53,19 +53,17 @@ def name(self) -> str:
     @property
     def available(self) -> bool:
         """Return True if entity is available."""
-        return self._api_enabled
+        return self._responsive
 
     @property
     def is_locked(self) -> bool:
         """Return True if the device is currently locked, else False."""
-        return not self._is_unlocked
+        return self._is_locked
 
     @property
     def state(self) -> str:
         """Get the state of the device."""
-        if self._is_unlocked:
-            return STATE_UNLOCKED
-        return STATE_LOCKED
+        return STATE_LOCKED if self._is_locked else STATE_UNLOCKED
```

Comment:
We shouldn't overwrite the `state` property. It's the responsibility of the base lock entity class.